.gitignore

```diff
@@ -1,2 +1,3 @@
 
 node_modules
+build
```

Comment:
@seemajune ah, so one caveat -- we have to keep the **build** folder because the `gh-pages` branch needs it (that's what is actually rendering the demo).